tests/Doctrine/Tests/Common/Annotations/Fixtures/IgnoredNamespaces/AnnotatedAtClassLevel.php

```diff
@@ -1,10 +1,12 @@
 <?php
 
 namespace Doctrine\Tests\Common\Annotations\Fixtures\IgnoredNamespaces;
+use Doctrine\Tests\Common\Annotations\Fixtures\Traits\AnnotateAtMethodLevelTrait;
 
 /**
  * @SomeClassAnnotationNamespace\Subnamespace\Name
  */
 class AnnotatedAtClassLevel
 {
+    use AnnotateAtMethodLevelTrait;
```

Comment:
Please use a new class to test these trait imports: don't modify existing assets